packages/utils/ast-utils.js

```diff
@@ -429,6 +429,62 @@ function addProperty(j, p, key, value) {
 	}
 	return;
 }
+
+/**
+ *
+ * Recursively updates an object/property to a node
+ * @param {any} j — jscodeshift API
+ * @param {Node} p - AST node
+ * @param {String} key - key of a key/val object
+ * @param {Any} value - Any type of object
+ * @returns {Node} - the updated ast
+ */
+
+function updateProperty(j, p, key, value) {
+	let valForNode;
```

Comment:
This can be moved after the `if`. There's no need to declare it if we exist from the function

Reply:
I think we can move it below `if (!p) { return; }` check, but not inside if block as it's getting assigned in every _if else_ condition which will make variable declarations redundant.

Reply:
> I think we can move it below if (!p) { return; } check
Yes, that's what I meant

Reply:
First rule of recursion is to have an exit condition, but for this function it needs to exit gracefully, as it's expected to return a node